go.mod

```diff
@@ -1,4 +1,6 @@
-go 1.19
+go 1.21
```

Comment:
Ok, I am going to close this PR and I will not backport the ics23 bump, since it's forcing a bump in the go version...